src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/nountable/DativeHandler.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright 2013
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package de.tudarmstadt.ukp.jwktl.parser.de.components.nountable;
+
+import de.tudarmstadt.ukp.jwktl.api.util.GrammaticalCase;
+
+public class DativeHandler extends CaseHandler {
+

```

Comment:
I am personally not a big fan of hundreds of tiny class files, so I would probably just instanciate CaseHandler with the pattern and the enum value within DEWordFormNounTableHandler. But of course this is a matter of taste.

Reply:
Disagree on this one. I think several classes are quite appropriate here. Right, we could pass pattern to the `CaseHandler`, but then we'd have to keep these patterns somewhere. They'd probably land as string constants in some constants class which will be huge and messy.